servicetalk-examples/http/defaultloadbalancer/src/main/java/io/servicetalk/examples/http/defaultloadbalancer/DefaultLoadBalancerClient.java

```diff
@@ -49,8 +51,49 @@ private static LoadBalancerFactory<InetSocketAddress, FilterableStreamingHttpLoa
+                        // - Round robin: linear iteration through the host list, skipping unhealthy hosts, per request.
+                        // - Power of two choices (P2C): randomly select two hosts and take the best based on score.
+                        // Both policies consider outlier detection (see below) but only P2C can bias traffic toward
+                        // more performant hosts. It does this by tracking request latency per-host using an
+                        // exponentially weighted moving average (EWMA) and using that combined with outstanding
+                        // request count to score hosts. The net result is typically a traffic distribution that will
+                        // show a preference toward faster hosts while also rapidly adjust to changes in backend
+                        // performance.
+                        new P2CLoadBalancingPolicy.Builder()
+                                // Set the max effort (default: 5). This is the number of times P2C will pick a random
+                                // pair of hosts in search of a healthy host before giving up. When it gives up it will
+                                // either attempt to use one of the hosts regardless of status if `failOpen == true` or
+                                // return a `NoActiveHosts` exception if failOpen == false.
+                                .maxEffort(6)
+                                // Whether to try to use a host regardless of health status (default: false)
+                                .failOpen(true)
+                                .build())
                 .outlierDetectorFactory(new XdsOutlierDetectorFactory<>(
-                        new OutlierDetectorConfig.Builder().build()
+                        // xDS compatible outlier detection has a number of tuning knobs. There are multiple detection
+                        // algorithms describe in more detail below. In addition to the limits appropriate to each
+                        // algorithm there are also parameters to tune the chances of enforcement when a particular
+                        // threshold is hit, the general intervals for periodic detection, penalty durations, etc.
+                        // See the `OutlierDetectorConfig` documentation for specific details.
+                        // The three primary detection mechanisms are:
+                        // - consecutive 5xx: Immediately mark a host as unhealthy if the specified number of failures
+                        //     happen in a row (subject to constraints). Good for rapid detection of failing hosts.
+                        // - success rate: statistical outlier detection pattern that runs on the detection interval.
+                        //     Good for true outlier detection but not as easy to reason about as failure percentage.
+                        // - failure percentage: Set a static limit on the percentage of requests that can fail before a
```

Comment:
Consider adding a brief not of what constitutes a failure (just exceptions? does this overlap with the 5xx consecutive failures?